chrome_extension/view_messages.py

```diff
@@ -0,0 +1,66 @@
+"""nicegui app to view messages from the native chrome extension."""
```

Comment:
What do you think about moving this to the `app` directory?

Also, can you plase clarify whether it provides any additional functionality compared to the HTML visualization?

Reply:
This file is not required anymore. It was removed.